pytube/cli.py

```diff
@@ -115,22 +124,34 @@ def _parse_args(
         dest="verbosity",
         help="Verbosity level, use up to 4 to increase logging -vvvv",
     )
+    parser.add_argument(
+        "--logfile",
+        action="store",
+        help="logging debug and error messages into a log file",
+    )
     parser.add_argument(
         "--build-playback-report",
         action="store_true",
         help="Save the html and js to disk",
     )
+    # TODO(nficano / yvain): empty caption code is not working
```

Comment:
can you make an issue corresponding to this and put the ticket number in the todo

Reply:
Actually, I can remove this comment since I fixed that as part of my edits